### ISSUE TITLE
paper-drawer-panel accessibility report 

### ISSUE THREAD

**Issue by lpalmaro** [PolymerElements/paper-drawer-panel#8]
Here are the accessibility developer tools audit results: 

[Warning] Text elements should have a reasonable contrast ratio (1)Not applicable tests (12)Passing tests (4)

IMO, the general contrast is sufficient. It may help to make the text on the button more bold though. 

There is something interesting that happens on the demo page when zooming in. If I press command + on my Mac to zoom in on the browser content, the layout changes. After a few zooms in, I see another button: Toggle drawer. This is a confusing experience, since this button did not apear before zooming. Also, when I then try to use the keyboard to untoggle the drawer, I am not able to. I press enter, and the gray overlay does not go away. When the gray overlay is over the button text, that contrast then becomes very poor.

**Comment by blasten** [comment]
@lpalmaro you see the toggle button because the `clientWidth` decreases when you zoomed in. That action (command +) will trigger the narrow mode once the threshold is reached.

**Comment by lpalmaro** [comment]
SG in regards to the appearance of the button. 

Please make sure it's possible to untoggle with the keyboard though.

**Comment by blasten** [comment]
If you focus the toggle button (from a tab keypress) and press enter, you can toggle the drawer. I will close this issue as we addressed all the feedback, but feel free to open it again if there's something missed.

**Comment by lpalmaro** [comment]
Contrast and keyboard navigation are great now. That said, it's still really tough to see focus state on these buttons. Is there anything we can do to improve this? I can't see the focus state at all.
